pytorch_lightning/utilities/xla_device_utils.py

```diff
@@ -0,0 +1,57 @@
+import functools
+import importlib
+from multiprocessing import Process, Queue
+
+TORCHXLA_AVAILABLE = importlib.util.find_spec("torch_xla") is not None
```

Comment:
What is the benefit of this over catching the try and catch the import error?

Reply:
This is a cleaner way to check if `torch_xla` exists. An advantage over try/catch would be that the `ImportError` could be raised from within the module itself, and try/catch would interpret that as `torch_xla` is not available. This method specifically checks for `torch_xla`.

Reply:
this is the way @awaelchli proposes...